tests/unit/test-firefox/test.firefox.js

```diff
@@ -324,6 +324,149 @@ describe('firefox', () => {
       }
     ));
 
+    it('configures a named profile', () => withTempDir(
+      (tmpDir) => {
+        const configureProfile =
+          sinon.spy((profile) => Promise.resolve(profile));
+        const app = 'fennec';
+        const profilePath = tmpDir.path();
+        const profileFinder = () => {
+          return {
+            getPath: sinon.spy((pathToProfile) =>
+                                    Promise.resolve(pathToProfile)),
+            hasProfileName: () => Promise.resolve(true),
+          };
+        };
+        const spy = sinon.spy(profileFinder, "getPath");
```

Comment:
@harikishen this is probably a leftover, but it is making the CI to fail on the eslint checks and it never reaches the unit tests:

- https://travis-ci.org/mozilla/web-ext/builds/244601162#L1351-L1353